src/vmware/azext_vmware/commands.py

```diff
@@ -80,3 +81,9 @@ def load_command_table(self, _):
         g.custom_show_command('show', 'addon_srm_show')
         g.custom_command('update', 'addon_srm_update')
         g.custom_command('delete', 'addon_srm_delete')
+
+    with self.command_group('vmware global-reach-connection', vmware_sdk, client_factory=cf_vmware) as g:
+        g.custom_command('create', 'globalreachconnection_create')
+        g.custom_command('list', 'globalreachconnection_list')
+        g.custom_command('delete', 'globalreachconnection_delete')

```

Comment:
what about add confirmation for delete command?

Reply:
I'm going to assume no. I've asked the PM. The only delete that confirms right now is `private-cloud`. If the PM comes back with a different answer, I'll put in another PR.